pynumdiff/optimize/_optimize.py

```diff
@@ -95,14 +100,14 @@
                    'gamma': (1e-3, 1000),
              'window_size': (15, 1000)})
 } # Methods with nonunique parameter sets are aliased in the dictionary below
-for method in [second_order, fourth_order]:
+for method in [second_order, fourth_order]: # Deprecated, redundant methods
```

Comment:
I realized basically all these methods I'm aliasing down here are redundant. An argument to truly consider them deprecated and remove them eventually.